settings.gradle.kts

```diff
@@ -64,7 +64,7 @@ include(":hedera-node:hedera-util-service")
 include(":hedera-node:hedera-util-service-impl")
 include(":hedera-node:hapi-utils")
 include(":hedera-node:hapi-fees")
-// include(":hedera-node:hapi")
+include(":hedera-node:hapi")
```

Comment:
I ask myself if `hapi` is really a good name for this module. I assume hapi-SOMETHING will be better?

Reply:
I am not sure, this is the true HAPI aka (Hedera API) it is the Hedera protobuf represented as Java API. The intent is it will be the only representation of HAPI in services. Do you have a better name?